apache-rat-plugin/src/main/java/org/apache/rat/mp/HeaderMatcherSpecification.java

```diff
@@ -22,7 +22,9 @@
 /**
  * Used to specify additional license matchers. A license matcher is basically a class, which implements
  * {@link org.apache.rat.analysis.IHeaderMatcher}.
+ * @deprecated use Configuration file to define licenses
  */
+@Deprecated
 public class HeaderMatcherSpecification
```

Comment:
This class contains unused methods? Is it reflection as well @Claudenw ?

Reply:
Other methods in these classes are marked as unused as well - I assume it's all reflection and means missing tests ....